### DISCUSSION TITLE
Discord group

### DISCUSSION THREAD

**Discussion by thewh1teagle** [k2-fsa/sherpa-onnx#1208] [Ideas]
Can you create discord group in addition to wechat?
We chat is popular mostly in China but it doesn't work well in other places but discord is more world wide.
Personally I can't even register WeChat.

I assume that discord should work in China?

**Comment by csukuangfj** [comment]
Just created one
https://discord.gg/qU3duDsH

Note that users would get a faster reply if the post issues on GitHub.

**Reply by eschmidbauer** [reply]
that link is invalid. Can someone share another discord link?

**Reply by manickavela29** [reply]
yes the link is invalid, @csukuangfj could you reshare the link

**Reply by thewh1teagle** [reply]
https://discord.com/channels/1269137500585328754/1269137500585328756

**Reply by manickavela29** [reply]
I see the msg, 

- you find yourself in strange space, you don't have access to any text channels or there are non in the server

**Reply by eschmidbauer** [reply]
![image](https://github.com/user-attachments/assets/dca9b0a2-ed16-43af-9e2c-b7822ede4036)
can someone share an invite link?

**Comment by csukuangfj** [comment]
Please use

https://discord.gg/TKFFznuD

**Comment by csukuangfj** [comment]
<img width="298" alt="Screenshot 2024-09-13 at 14 55 50" src="https://github.com/user-attachments/assets/c6de3525-aff7-49d3-9a74-e4a70f6f5a3d">
